src/app/home/work-item-widget/work-item-widget.component.html

```diff
@@ -1,44 +1,58 @@
 <div class="create-work-item-widget card-pf card-f8">
   <div class="card-pf-heading-bg">
     <div class="card-pf-heading">
-      <h2 class="card-pf-title">My work items</h2>
+      <h2 class="card-pf-title">My work items
+      <span *ngIf="spaces.length !== 0">
```

Comment:
@dlabrecq I worked with @catrobson on the update this evening - trying to save some spacing. I didn't have that overlap problem that you are showing here - what viewport resolution are you viewing this with?